cap_client/api/analysis_api.py

```diff
@@ -35,16 +35,24 @@
 
 class AnalysisAPI(CapAPI):
     """Interface for CAP analysis methods."""
-    def get_drafts(self, all=False):
+    def get_drafts(self, all=False, type=None):
         """Get list of user's draft analyses.
 
         :param all: show all (not only created by user)
         :type all: bool
+        :param type: type of analysis
+        :type type: string
         :return: list of analyses
         :rtype: list
         """
+        url = 'deposits/?q='
+        if type:
```

Comment:
We can use `urlencode` to better construct the query string needed. It is easier and safer to use